packages/onchainkit/scripts/get-next-version.js

```diff
@@ -2,37 +2,58 @@ import { execSync } from 'child_process';
 import process from 'process';
 import fs from 'fs';
 import path from 'path';
+import { fileURLToPath } from 'url';
 
 const packageName = '@coinbase/onchainkit';
+const onchainkitPath = 'packages/onchainkit';
 
 function getNextVersion() {
+  const currentFilePath = fileURLToPath(import.meta.url);
+  const currentDir = path.dirname(currentFilePath);
+
+  // Move to monorepo root
+  const monorepoRoot = path.resolve(currentDir, '../../..');
+  process.chdir(monorepoRoot);
+
+  let nextVersion = '';
+
   try {
-    process.chdir('../..');
-    const output = execSync(
-      'pnpm changeset status --verbose --since=origin/main',
-      {
-        encoding: 'utf-8',
-      },
-    );
+    // This throws an error if there are no changes in any packages
+    const output = execSync('pnpm changeset status --verbose', {
+      encoding: 'utf-8',
+    });
 
     const lines = output.split('\n');
     const onchainkitVersionLine = lines.find((line) =>
       line.includes(packageName),
     );
 
-    if (!onchainkitVersionLine) return;
+    if (!onchainkitVersionLine)
+      throw new Error('No onchainkit version line found');
```

Comment:
would have expected linter to complain about no brackets, but as long as it's passing lint i'm fine with it

Reply:
I kinda line one liners for these and the linter's fine with it, but if we wanna enforce brackets later we can turn that rule on and `--fix`